PreFreeSurfer/scripts/BrainExtraction_FNIRTbased.sh

```diff
@@ -8,69 +8,56 @@
 #  Usage Description Function
 # ------------------------------------------------------------------------------
 
-script_name=$(basename "${0}")
-
-Usage() {
-	cat <<EOF
-
-${script_name}: Tool for performing brain extraction using non-linear (FNIRT) results
-
-Usage: ${script_name}
-  [--workingdir=<working dir>]
-  --in=<input image>
-  [--ref=<reference highres image>]
-  [--refmask=<reference brain mask>]
-  [--ref2mm=<reference image 2mm>]
-  [--ref2mmmask=<reference brain mask 2mm>]
-  --outbrain=<output brain extracted image>
-  --outbrainmask=<output brain mask>
-  [--fnirtconfig=<fnirt config file>]
-
-EOF
-}
-
-# Allow script to return a Usage statement, before any other output or checking
-if [ "$#" = "0" ]; then
-    Usage
-    exit 1
-fi
+# script_name=$(basename "${0}")
```

Comment:
Same, and check the other files too, please.

Reply:
Done